config/locales/en.yml

```diff
@@ -59,6 +50,40 @@ en:
     show:
       title: "Your budget:"
 
+  budgets:
```

Comment:
Do the budget translations belong in this PR?

Reply:
No, but from memory I lost them in a merge conflict when I took the wrong choice, so I added them in so I didn't forget, do you want me to remove them and add them back in another PR?

Reply:
No worries 👍